test/tests/meshgenerators/xy_delaunay_generator/tests

```diff
@@ -1,6 +1,6 @@
 [Tests]
   design = 'meshgenerators/XYDelaunayGenerator.md'
-  issues = '#20192 #26776 #27545'
+  issues = '#20192 #26776 #27545 #27828'
```

Comment:
we should make one of the exodiffs use pedantic to make sure this does not regress. you ll need a regold